sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -124,10 +129,18 @@ def _parse_conn_str(
     # check that endpoint is valid
     if not endpoint:
         raise ValueError("Connection string is either blank or malformed.")
-    parsed = urlparse(endpoint)
-    if not parsed.netloc:
-        raise ValueError("Invalid Endpoint on the Connection String.")
-    host = cast(str, parsed.netloc.strip())
+    emulator = bool(use_emulator=="true") if use_emulator else False
+    if emulator:
+        if not _is_local_endpoint(endpoint):
+            raise ValueError("Invalid Endpoint on the Connection String. "
+                             "For Development Connection Strings, the Endpoint should be "
+                             "a localhost.")
+        host = endpoint   
```

Comment:
will endpoint be here in the format `localhost:9600` ?

Reply:
it will it will get url.parsed in the connection in pyamqp